tests/integration/_common.sh

```diff
@@ -3,24 +3,49 @@
 root=${root:-$(mktemp -d -t leitXXXX)}
 echo "Root integration tests directory: $root"
 config_dir="$root/conf"
-store_flags="--config-dir $config_dir --work-dir $root/work"
-store_flags="$store_flags --logs-dir $root/logs"
 tls_sni_01_port=5001
 http_01_port=5002
 sources="acme/,$(ls -dm certbot*/ | tr -d ' \n')"
-export root config_dir store_flags tls_sni_01_port http_01_port sources
+export root config_dir tls_sni_01_port http_01_port sources
+certbot_path="$(command -v certbot)"
+# Flags that are added here will be added to Certbot calls within
+# certbot_test_no_force_renew.
+other_flags="--config-dir $config_dir --work-dir $root/work"
+other_flags="$other_flags --logs-dir $root/logs"
 
 certbot_test () {
     certbot_test_no_force_renew \
         --renew-by-default \
         "$@"
 }
 
+# Checks that the Certbot version is at least the given version number. This is
```

Comment:
As bash function does not return anything, it fails or succeed, and its return code represents this state, with 0 being a succeed. To avoid confusion here, and reduce the comment size, you could say "Succeeds if the Certbot..., fails otherwise." as the first sentence, and remove the last one.